test/Integration/IntegrationTestBase.cs

```diff
@@ -311,7 +387,7 @@ private void LogOutput(string output)
             }
         }
 
-        private DataReceivedEventHandler GetTestOutputHandler(int processId)
+        private DataReceivedEventHandler GetTestOutputHandler(string processId = null)
```

Comment:
Why are you not including the pid for the maven process?

And right now this would print out `[] <data here>` for those messages, so even if we did want to allow this to skip the ID then we should also be skipping adding the PID to the message output. (but I think it'd be best to just include the PID for all of them anyways)